README.md

```diff
@@ -112,7 +112,9 @@ class BadInstagramCloneApp extends Component {
   }
 
   takePicture() {
-    this.camera.capture()
+    const options = {};
+    //options.location = ...
+    this.camera.capture({metadata: options})
```

Comment:
The existing _readme.md_ says

> metadata This is metadata to be added to the captured image.
> location This is the object returned from navigator.geolocation.getCurrentPosition() (React Native's geolocation polyfill). It will add GPS metadata to the image.

So, for example the shape if you want to add location info would be:

`this.camera.capture({metadata: {location: {lat:.., lon: ...}})`

This PR doesn't change that.